shared/lib-camel-connector/src/main/java/gov/va/vro/camel/MessageQueueConfiguration.java

```diff
@@ -28,6 +32,12 @@ public class MessageQueueConfiguration {
   private final String BIE_EVENTS_CONTENTION_COMPLETED = "bie-events-contention-completed";
   private final String BIE_EVENTS_CONTENTION_DELETED = "bie-events-contention-deleted";
   private final String SAVE_TO_DB_PREFIX = "saveToDB-";
+  private final String QUEUE_MESSAGES_DLQ = "bie-events-dql";
+  private final String DLX_EXCHANGE_MESSAGES = "bie-events.dlx";
```

Comment:
Is there any reason we use the `.` instead of `-` for the exchange name here?

Reply:
I was using the same standard I saw in some RabbitMQ example code. I don't think it has any functional purpose